core/build.gradle.kts

```diff
@@ -25,6 +25,10 @@ android {
       isMinifyEnabled = false
     }
   }
+
+  defaultConfig {
+    vectorDrawables.useSupportLibrary = true
```

Comment:
this shouldn't be necessary, I know AS complains but we absolutely are applying this via our build plugin